libexec/scoop-import.ps1

```diff
@@ -21,18 +24,25 @@ if (Test-Path $scoopfile) {
 
 if (!$import) { abort 'Input file not a valid JSON.' }
 
-$import.config.PSObject.Properties | ForEach-Object {
-    set_config $_.Name $_.Value | Out-Null
-    Write-Host "'$($_.Name)' has been set to '$($_.Value)'"
+foreach ($item in $import.config.PSObject.Properties) {
+    set_config $item.Name $item.Value | Out-Null
+    Write-Host "'$($item.Name)' has been set to '$($item.Value)'"
 }
 
-$import.buckets | ForEach-Object {
-    add_bucket $_.Name $_.Source | Out-Null
-    $bucket_names += $_.Name
+
+$local_buckets = Get-LocalBucket
+foreach($item in $import.buckets){
+    if($item.Name -in $local_buckets){
```

Comment:
As I said, this check is not sufficient, it only checks the names. The proper method would be to revert the commit and remove the message part `Use 'scoop bucket rm $name' to remove it.` from the `add_bucket` function in lib/buckets.ps1

Reply:
You're right. I will revert the commit.